backend/routers/issues.py

```
@@ -478,6 +478,37 @@ def subscribe_push_notifications(
         message="Push subscription created"
     )
 
+@router.get("/api/issues/user", response_model=List[IssueSummaryResponse])
+def get_user_issues(
+    user_email: str = Query(..., description="Email of the user"),
+    limit: int = Query(10, ge=1, le=50, description="Number of issues to return"),
+    offset: int = Query(0, ge=0, description="Number of issues to skip"),
+    db: Session = Depends(get_db)
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)